### DISCUSSION TITLE
Custom module SLA policy Yetiforce

### DISCUSSION THREAD

**Discussion by kseibi** [YetiForceCompany/YetiForceCRM#14707] [How to]
Hello, I am using Yetiforce 6.1, the only available module for SLA policy is "Tickets", how can we add custom made module to the SLA policy? could it be done through Mysql or it's which file has to be changed to add the other custom module?

**Comment by mariuszkrzaczkowski** [comment]
I don't know if it will work, but you can try it

```php
\App\RecordStatus::activate('HelpDesk','ticketstatus');
```
